src/components/strategies/common/BudgetInput.tsx

```diff
@@ -8,44 +8,62 @@ import { UseQueryResult } from '@tanstack/react-query';
 interface Props {
   id?: string;
   children?: ReactNode;
-  order: OrderCreate;
+  budgetValue: string;
+  budgetError?: string;
```

Comment:
Since the name of the component is `BudgetInput`, I think we can rename to `value` and `error`